### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by SteffenHeu**
- fix compound name matching in ms2 quality and library export
- fix cast exception in msms mirror
- add green-yellow paint scale as default (better contrast with black heatmap background)
- [add correct module during ScanFilteringTask](https://github.com/mzmine/mzmine3/pull/991/commits/752740377605ae3604a58cede9a553b3a20b8949)

### CODE REVIEW

File: src/main/java/io/github/mzmine/parameters/parametertypes/paintscale/PaintScalePaletteParameter.java [hunk 1]
```diff
@@ -106,11 +107,16 @@ public void loadValueFromXML(Element xmlElement) {
 
     selected = (selected != -1) ? selected : 0;
 
+    if (!palettes.contains(SimpleColorPalette.GREEN_YELLOW)) {
+      palettes.add(SimpleColorPalette.GREEN_YELLOW);
```
Comment (robinschmid) [inline src/main/java/io/github/mzmine/parameters/parametertypes/paintscale/PaintScalePaletteParameter.java:111]: As a default we should use someting that is colorblind friendly.
I think the default here is usually blue-red, blue to yellow, blue, to brown. Something like that. 

green, yellow, red is not so easy

Comment (robinschmid) [reply src/main/java/io/github/mzmine/parameters/parametertypes/paintscale/PaintScalePaletteParameter.java:111]: Original colors
image

Protanopia - colorblindness
image

Comment (robinschmid) [reply src/main/java/io/github/mzmine/parameters/parametertypes/paintscale/PaintScalePaletteParameter.java:111]: In this case green yellow is like a monochrome scale. Only the dark/light are different

**Comment by SteffenHeu** [comment]
grafik
new default palette

**Comment by robinschmid** [comment]
looks great

**Pull request merged**